follower/upstream_connector.go

```diff
@@ -0,0 +1,112 @@
+package follower
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/sethvargo/go-retry"
+	"go.uber.org/atomic"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/lifecycle"
+	"github.com/onflow/flow-go/network/p2p"
+)
+
+// upstreamConnector tries to connect the unstaked AN with atleast one of the configured bootstrap access nodes
+type upstreamConnector struct {
```

Comment:
It is a reasonable request, indeed. Rather than this I suggest putting it to a <root>/observer. Normally anything in cmd/ should not be the base of any other library as final code with main.
I prepare the change, and I let you criticize this opinion later.

Reply:
I put it to follower/upsteram_connector.go. Alternatively, it can go to somewhere like upstream/upsteram_connector.go.

Reply:
Fix: https://github.com/GetElastech/flow-go/commit/9fe0b71f614da0a7b3793e3a39a6374b53bccccd